.github/workflows/build.yml

```diff
@@ -1,25 +1,38 @@
 name: "Build and test Lua project"
+
 on: [push]
 jobs:
   build:
+    strategy:
+       fail-fast: true
+       matrix:
+          lua_version: [5.1, 5.2, 5.3, 5.4]
```

Comment:
Is that really necessary? Wouldn't it be enough to test highest and lowest version?

Reply:
@redcatbear: what is your opinion on this?

Reply:
His answer: yes, it's necessary. Lua sometimes has breaking API changes.